terracotta-kit/src/assemble/init/sysv/terracotta.properties

```diff
@@ -0,0 +1,25 @@
+# Configuration file for the Terracotta SysV init script
+# Intended to be placed into /etc/default/terracotta
+
+# Define some variables
+# Name of app
+APP=terracotta
+
+# Name of the user to run as
+USER=terracotta
+
```

Comment:
NAME=server1

Reply:
or NAME=server2

Reply:
is this meant to replace $APP or be a different config item?

Reply:
this is an addition